src/CMakeLists.txt

```diff
@@ -126,7 +137,10 @@ install(EXPORT
 target_link_libraries(${LIBFM_QT_LIBRARY_NAME}
     Qt5::Widgets
     Qt5::X11Extras
-    ${FM_LIBRARIES}
+    ${GLIB_LIBRARIES}
```

Comment:
Why aren't we using the `*_LDFLAGS` ? This caused problems with libraries installed in non-standard paths problems in other components in the past...

Reply:
@palinek our FindGLIB.cmake does not define LDFLAGS:
https://github.com/lxqt/lxqt-build-tools/blob/master/cmake/find-modules/FindGLIB.cmake

Reply:
Sorry,... forgot about that... I thought we're using just the pkg-config...